test/integration/044_run_operations_tests/test_run_operations.py

```diff
@@ -69,3 +69,7 @@ def test__postgres_select(self):
     @use_profile('postgres')
     def test__postgres_access_graph(self):
         self.run_operation('log_graph')
+
+    @use_profile('postgres')
+    def test__postgres_print(self):
```

Comment:
I'd like a comment on this test describing precisely what we're testing. In this case I think it's just checking that calling the print macro doesn't blow up, which imo is good enough test coverage for me.

Reply:
Agree that a comment here would be helpful.

Reply:
Added!